Mod2/Lessons/Week1/VersionControlAndSoloGit.md

```diff
@@ -0,0 +1,113 @@
+# Version Control with Git/Github
+
+## Learning Goals
+- Define Version Control and its benefit
+- Build a mental model of branching workflow
+- Practice committing and branching on a solo repository
+
+## Warm Up
+In a notebook, reflect on the following:
+* When doing something like writing an essay, or research paper, how have you saved your work?
+* For school projects, what tools have you used to collaborate on a presentation or paper?
+* What do you already know about git and GitHub?
```

Comment:
I'm sort of torn about these questions especially for mod 1 week 2 because we don't actually expect they know anything already and I think they can sometimes come across as expecting you to already know things. Words are hard, but maybe a rephrase to something like

If you've already heard of git and Github, what do you already know?

or

If you've already heard of git and Github, what do you already know? (It's also totally ok if these terms are brand new!)

Reply:
Oh wait, this is mod 2, feel better about this question if that's the case. How do they turn things in and such mod 1 without Git?

Reply:
Talked with Megan, I like the idea of a tiny git lesson Mod 1 with just fork, clone, commit and push. With no expectation of understanding what's happening, just the procedural skills required to turn things in and get starter code.

Reply:
@rtillies is a little git lesson part of what you're working on for the VSCode/repl.it unit 1 rework? If not maybe we make a ticket for the project board?

Reply:
I like that idea a lot @memcmahon !